dask_sql/physical/utils/ml_classes.py

```diff
@@ -0,0 +1,122 @@
+def get_cpu_classes():
+    try:
+        from sklearn.utils import all_estimators
+
+        cpu_classes = {
+            k: v.__module__ + "." + v.__qualname__ for k, v in all_estimators()
+        }
+    except ImportError:
+        cpu_classes = {}
+
+    cpu_classes = add_boosting_classes(cpu_classes)
+
+    return cpu_classes
+
+
+def get_gpu_classes():
+    gpu_classes = {
+        # cuml.dask
+        "DBSCAN": "cuml.dask.cluster.dbscan.DBSCAN",
+        "KMeans": "cuml.dask.cluster.kmeans.KMeans",
+        "PCA": "cuml.dask.decomposition.pca.PCA",
+        "TruncatedSVD": "cuml.dask.decomposition.tsvd.TruncatedSVD",
+        "RandomForestClassifier": "cuml.dask.ensemble.randomforestclassifier.RandomForestClassifier",
+        "RandomForestRegressor": "cuml.dask.ensemble.randomforestregressor.RandomForestRegressor",
+        # ImportError: dask-glm >= 0.2.1.dev was not found, please install it to use multi-GPU logistic regression.
+        # "LogisticRegression": "cuml.dask.extended.linear_model.logistic_regression.LogisticRegression",
```

Comment:
Sure, I opened https://github.com/dask-contrib/dask-sql/issues/1015 and linked to it.

Reply:
Not sure if this is something we should fix on the Dask-SQL side (see https://github.com/dask-contrib/dask-sql/issues/1015#issuecomment-1409076884). Would it make sense to have some sort of try/except logic around this?

Reply:
Thanks @sarahyurick 🙂 with that additional context, IMO I feel like the best option on our end for now would be to include `cuml.dask.extended.linear_model.logistic_regression.LogisticRegression` in the mapping and raise the `ImportError` directly,  with the idea being that users would just subsequently install `dask-glm` into their environment and try rerunning their query.

I suppose if we wanted to be as informative as possible, we could reraise the error with some addendum about using the non-Dask equivalent class if installing `dask-glm` isn't an option, though I think that message would probably make more sense to add upstream since it would be informative to cuML users at large.